docs/src/components/menu/index.js

```diff
@@ -62,7 +63,7 @@ export default class Menu extends Component {
     return (
       <Drawer.TemporaryDrawer ref={drawer => (this.drawer = drawer)}>
         <Drawer.TemporaryDrawerHeader className="mdc-theme--primary-bg">
-          Components
+          Components @ (v{json.devDependencies["preact-material-components"]})
```

Comment:
Nice one! Shouldn't `json-loader` be configured as a module in `webpack` though?